modules/local/virsorter2/main.nf

```diff
@@ -20,24 +21,20 @@ process VIRSORTER2 {
   script:
   def args = task.ext.args ?: ''
   """
-  # Settings to speed up hmmsearch
-  # TODO: this needs to be tested, it doesn't seem to speed up so we decided to chunk the fasta instead
-  # virsorter config --set HMMSEARCH_THREADS=4
-  # virsorter config --set FAA_BP_PER_SPLIT=50000
-  
-  # extract chunk number to rename output files
+  # Extract chunk number to rename output files
   filename=\$(basename ${fasta})
   # Extract VALUE (assuming the filename is in format NAME.VALUE.fasta)
   export value="all"
   value=\$(echo "\$filename" | cut -d'.' -f2)
 
-  virsorter run \
-            --db-dir ${database} \
-            -w virsorter2 \
-            -i ${fasta} \
-            -j ${task.cpus} \
-            ${args} \
-            all
+  virsorter run \\
+    --db-dir ${database} \\
+    --use-conda-off \\
```

Comment:
That argument is sitting in modules.config already https://github.com/EBI-Metagenomics/emg-viral-pipeline/blob/master/configs/modules.config#L684. I think that should be there.
That also reminded me that we probably should add a correct condition and do not include if pipeline will be launched with conda. There is a note about it in modules.config and ofc I forgot about it

Reply:
I overlooked that one, it should be there.. that's fine. 
With conda .. I think we would just need to add https://github.com/EBI-Metagenomics/containers/blob/master/virsorter2/2.2.4/env.yaml as `conda env.yml` and we can keep the `use-conda-off`.

Reply:
I'll keep the one in args.. and remove the on in the module.

Reply:
Done :)